test/test.dart

```diff
@@ -249,6 +246,7 @@ void testAppGoldens(
   bool? skip,
   Object? tags = _defaultTagObject,
 }) {
+  EditableText.debugDeterministicCursor = true;
```

Comment:
This should copy the current value of it and apply it after the test ends (use try/finally for that)